src/pages/Scheduling/ScheduleExceptions.tsx

```diff
@@ -44,15 +47,36 @@ export default function ScheduleExceptions({
 }: Props) {
   const { t } = useTranslation();
 
+  const { data: isSchedulableResource } = useQuery({
+    queryKey: ["is_schedulable_resource", facilityId, userId],
+    queryFn: query(scheduleApis.appointments.availableUsers, {
+      pathParams: { facility_id: facilityId },
+    }),
+    select: (data: { users: UserBase[] }) =>
+      data.users.some(({ id }) => userId === id),
+  });
+
   if (items == null) {
     return <Loading />;
   }
 
-  if (items.length === 0) {
+  const getMessage = () => {
+    if (!isSchedulableResource) {
+      return t("exceptions_can_be_created_only_after_creating_a_schedule");
+    }
+    if (items.length === 0) {
+      return t("no_scheduled_exceptions_found");
+    }
+    return null;
+  };
+
+  const message = getMessage();
+
+  if (message) {
     return (
       <div className="flex flex-col items-center text-center text-gray-500 py-16">
         <CareIcon icon="l-calendar-slash" className="size-10 mb-3" />
-        <p>{t("no_scheduled_exceptions_found")}</p>
+        <p>{message}</p>
```

Comment:
instead of doing this, we can make the code more readable like this:

```tsx
if (!isSchedulableResource) {
  return <div ...>
    ...
    {t("exceptions_can_be_created_only_after_creating_a_schedule")}
  </div>
}

if (items.length === 0) {
  return <div ...> same as old one...</div>
}
...
```